.inshpect.toml

```diff
@@ -133,6 +133,8 @@ patterns = [
     { pattern = '(\bpika::\b)?\butil::\bresult_of\b', replacement = 'std::invoke_result' },
     { pattern = '(\bpika::\b)?\butil::\bdecay\b', replacement = 'std::decay' },
     { pattern = '(\bpika::\b)?\butil::\bdecay_t\b', replacement = 'std::decay_t' },
+    { pattern = '(\bpika::\b)?\butil::\bdetail::\binvoke_result\b', replacement = 'std::invoke_result' },
+    { pattern = '(\bpika::\b)?\butil::\bdetail::\binvoke_result_t\b', replacement = 'std::invoke_result_t' },
```

Comment:
I think we can leave these out since they don't exist anymore (yes, I see there are others that we could remove as well above... but you don't need to do those as well).